test/back2back_test.py

```diff
@@ -125,6 +125,66 @@ def test_dlc_less_than_eight(self):
         msg = can.Message(is_extended_id=False, arbitration_id=0x300, data=[4, 5, 6])
         self._send_and_receive(msg)
 
+    def test_message_direction(self):
+        # Verify that own message received has is_rx set to False while message
+        # received on the other virtual interfaces have is_rx set to True
+        if self.INTERFACE_1 != "virtual":
+            raise unittest.SkipTest(
+                "Message direction not yet implemented for socketcan"
+            )
+        bus3 = can.Bus(
+            channel=self.CHANNEL_2,
+            bustype=self.INTERFACE_2,
+            bitrate=self.BITRATE,
+            fd=TEST_CAN_FD,
+            single_handle=True,
+            receive_own_messages=True,
+        )
+        try:
+            msg = can.Message(
+                is_extended_id=False, arbitration_id=0x300, data=[2, 1, 3]
+            )
+            bus3.send(msg)
+            recv_msg_bus1 = self.bus1.recv(self.TIMEOUT)
+            recv_msg_bus2 = self.bus2.recv(self.TIMEOUT)
+            self_recv_msg_bus3 = bus3.recv(self.TIMEOUT)
+
+            self.assertEqual(recv_msg_bus1.is_rx, True)
+            self.assertEqual(recv_msg_bus2.is_rx, True)
+            self.assertEqual(self_recv_msg_bus3.is_rx, False)
```

Comment:
Please simplify by using assertTrue/assertFalse

```suggestion
            self.assertTrue(recv_msg_bus1.is_rx)
            self.assertTrue(recv_msg_bus2.is_rx)
            self.assertFalse(self_recv_msg_bus3.is_rx)
```